src/content/self-hosted/guides/okta/index.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Use Okta as your Authentication Provider
+description: Guide on how use Okta as the authentication provider of your Okteto Self-hosted instance
+id: identity-provider-okta
+---
+
+import Image from '@theme/Image';
+import TiersList from '@theme/TiersList';
+
+#  Use Okta as your Identity Provider<TiersList tiers="Self-Hosted" />
+
+This guide will walk you through the steps necessary to configure Okta as the authentication provider for your instace of Okteto. 
+
+Please refer to [Okta's official documentation](https://developer.okta.com/docs/concepts/oauth-openid/) on Open ID Connect for more information.
+
+## Prerequisites
+
+- An Instance of [Okteto](https://okteto.com/docs/self-hosted/).
+- Helm 3.0+ installed in your local machine.
+- Access to an Okta account with permissions to create Applications.
+
+## Create the Okta Application
+
+Create a new app integration, using the values below:
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/new-app.png" alt="Create the new app" /></p>
+
+Then, fill in the following values on the “New Web App Integration” window:
+
+1. Name: The name of your app (e.g. Okteto)
+2. Grant type: authorization code
+3. Sign In Redirect: https://okteto.{YOUR_DOMAIN}/auth/callback
+4. Assignments:  We recommend you create a dedicated group for Okteto, so you can control who can access your instance from Okta.
+
+Leave all the other values empty, and press the save button.
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/new-web-app-integration.png" alt="Create the new app integration" /></p>
```

Comment:
the redirect url in the image says `https://demo.okteto.dev/authorization-code/callback`, but should it be `https://demo.okteto.dev/authorization/callback`?

Reply:
ah good one, I'll update it